src/implementations/twig/components/notification/notification.html.twig

```diff
@@ -47,31 +47,34 @@
 
 {# Print the result #}
 
-<div class="{{ _css_class }}" data-ecl-message role="alert"{{ _extra_attributes|raw }}>
+<div class="{{ _css_class }}" data-ecl-notification role="alert"{{ _extra_attributes|raw }}>
   {% include '@ecl/icon/icon.html.twig' with {
     icon: _icon,
-    extra_classes: 'ecl-message__icon',
+    extra_classes: 'ecl-notification__icon',
   } only %}
   
-  <div class="ecl-message__content">
+  <div class="ecl-notification__content">
     {% if _close is not empty %}
-      {% set _close_icon = {} %}
+      {% set _close_icon = { name: 'close-filled' } %}
```

Comment:
this is really confusing, we first set the _close_icon as an object with a name property, then if the icon is present in the data we just override that with what is coming from the data.
I'm not sure what is the desired behaviour, should the users be able to use a different icon if they want? In that case this approach can work but i would still expect a merge, so that if the name is not present in the data we would complete that missing info, but the icon we set here is different than the one requested in the data and used in the demo, here we have close-filled and we demo close, instead.
If the icon is not supposed to be overridden, then the merge should happen the other way around, and we would force the name of the icon that way. To be honest i would expect also the size of the icon to be set by the template, in other words we only  need a path to an icon sprite, most likely, and we could even think about only asking for that and building the icon object ourselves

Reply:
yes, I was not sure how to handle this... Having different icon for both systems would mean relying on the user to provide the correct data. I put a line for that in the documentation, we'll see if it comes as an issue during the first implementation (that could force the designers to review their work)
What I could do for now is having different data files for EC and EU, so we don't have to do this kind of weird thing in the template or story.